gobblin-compaction/src/test/java/org/apache/gobblin/compaction/mapreduce/AvroCompactionTaskTest.java

```diff
@@ -173,6 +173,40 @@ public void testAvroRecompaction() throws Exception {
     Assert.assertTrue(fs.exists(new Path (basePath, "Identity/MemberAccount/hourly/2017/04/03/10")));
   }
 
+  @Test
+  public void testAvroRecompactionWriteToNewPath() throws Exception {
+    FileSystem fs = getFileSystem();
+    String basePath = "/tmp/testRecompaction";
+    fs.delete(new Path(basePath), true);
```

Comment:
Instead of deleting the path inside the test, can we set java.io.File#deleteOnExit(true) for the base dir? It seems like the base dir is being used in other tests and if the tests are executed in multiple threads, we can get unpredictable behavior.

Reply:
In this case, we want to make sure the path is clean before start test, I rename the base path to another one to avoid conflict